reorder-journal.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v hledger)" ]; then
+  echo 'Error: hledger is not installed.' >&2
+  exit 1
+fi
+
+SEPARATOR=";;;;;;;;;;;;;;;;;;;;;;;;;;;;;"
+LEDGER_FILE="$1"
+
+if [[ -z "$LEDGER_FILE" ]]; then
+  echo "Please specify the ledger file in the first argument. ./reorder-journal.sh hledger.journal"
+  exit 1
+fi
+
+TMPFILE_SORTED=$(mktemp /tmp/hledger.XXXXXX)
+TMPFILE_SKELETON=$(mktemp /tmp/hledger.XXXXXX)
+
+if ! hledger print -f "$LEDGER_FILE" > "$TMPFILE_SORTED";
+then
+  rm "$TMPFILE_SORTED" "$TMPFILE_SKELETON"
+  echo "The ledger file is not valid. Run `hledger print` to see the cause of the error."
```

Comment:
(assuming bash will like the backtick, otherwise lets change to `'`)

Reply:
Correct insight, bash doesn't like it.